code/modules/mob/living/living.dm

```diff
@@ -1,3 +1,5 @@
+var/blood_color
+
```

Comment:
Get rid of this, we don't need it as a global variable as it is already defined on the borgi itself